master/pkg/model/experiment.go

```diff
@@ -463,3 +463,74 @@ type SearcherEvent struct {
 	EventType    string  `db:"event_type"`
 	Content      JSONObj `db:"content"`
 }
+
+// MetricType denotes what type of step (training / validation) a metric is from.
+type MetricType int
+
+const (
+	// TrainingMetric designates metrics from training steps.
+	TrainingMetric MetricType = iota
+	// ValidationMetric designates metrics from validation steps.
+	ValidationMetric MetricType = iota
```

Comment:
```suggestion
	ValidationMetric MetricType
```

Reply:
Personally I prefer it on each line. Functionally no different, and just clear, IMO.